app/src/main/res/layout/loyalty_card_view_layout.xml

```diff
@@ -187,7 +206,30 @@
                 android:contentDescription="@string/setBarcodeHeight"
                 android:max="100" />
         </LinearLayout>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toTopOf="@+id/fullscreen_button_minimize"
+            android:layout_marginBottom="25dp"
+            android:layout_marginStart="15.0dip"
+            android:layout_marginEnd="15.0dip">
+
+            <TextView
+                android:importantForAccessibility="no"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:text="@string/width"/>
 
+            <SeekBar
+                android:id="@+id/barcode_widthscaler"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:contentDescription="@string/setBarcodeWidth"
+                android:max="100" />
```

Comment:
This seekbar doesn't seem to use the theming colour like the Height bar does:
![image](https://github.com/user-attachments/assets/b2985316-d36c-4fcb-aea6-56712f807e96)